test/test_public_bindings.py

```diff
@@ -250,6 +250,7 @@ def test_modules_can_be_imported(self):
             "torch._inductor.codegen.cuda.cuda_kernel",
             "torch.onnx._internal.fx._pass",
             "torch.onnx._internal.fx.analysis",
+            "torch.onnx._internal.fx.decomposition_skip",
```

Comment:
It would be best to make this module not crash on import.

Reply:
@albanD could you approve? Please let me know if this is a concern.